yarn-project/sequencer-client/src/global_variable_builder/config.ts

```diff
@@ -0,0 +1,19 @@
+import { EthAddress } from '@aztec/circuits.js';
+
+/**
+ * Configuration of the L1GlobalReader.
+ */
+export interface GlobalReaderConfig {
+  /**
+   * Rollup contract address.
+   */
+  rollupContract: EthAddress;
+  /**
+   * The RPC Url of the ethereum host.
+   */
+  rpcUrl: string;
+  /**
+   * The API key of the ethereum host.
+   */
+  apiKey?: string;
```

Comment:
isn't this on sequencer config already?

Reply:
Yes, so it is just to not require the entire config to be passed to the builder. If you look at the other configs, you will see similar with values that are "shared" between them, so the full config makes up the union of them.